src/types.ts

```diff
@@ -1,14 +1,17 @@
+import { DiagLogger, DiagLogLevel } from '@opentelemetry/api';
 import { NodeSDKConfiguration } from '@opentelemetry/sdk-node';
 
 /** Lightstep specific configuration options */
 export interface LightstepConfigType {
   accessToken?: string;
-  metricsHostEnabled?: boolean;
+  // metricsHostEnabled?: boolean;
```

Comment:
Are we keeping this for when we re-enable metrics, or should we remove it. If we want to remove it, we should remove `metricEndpoint` below.

Reply:
at first I wanted to keep it, but the history is in git anyway, removing.